tests/models/opt/test_tokenization_opt.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2020 The HuggingFace Team. All rights reserved.
```

Comment:
Let's delete this

Reply:
resolved in 1e21983a4ed08ca2c91d2d218e5048ba50998efa